CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ### Features
 ### Enhancements
 ### Bug Fixes
+Fixed exception for case when Hybrid query being wrapped into bool query ([#490](https://github.com/opensearch-project/neural-search/pull/490)
```

Comment:
Do we also have a task to not allow such queries in the future?

Reply:
Yes, this is the general plan, but there are cases when it's not that easy. For instance, when nested field is present in the index mapping then the query will be wrapped into the Bool query where hybrid will be one of the child clause. We're investigating possible options.

Reply:
@martin-gaievski can you link the github issue here?

Reply:
sure, this is a link to an issue https://github.com/opensearch-project/neural-search/issues/466